naverland.net,"HTTPSConnectionPool(host='naverland.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bdf7c670>, 'Connection to naverland.net timed out. (connect timeout=2)'))","HTTPConnectionPool(host='naverland.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f62bdf3cc40>, 'Connection to naverland.net timed out. (connect timeout=2)'))"
nb.rs,"HTTPSConnectionPool(host='nb.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nb.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))"
ncc.gov.ng,"HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)"
nearfinderza.com,"HTTPSConnectionPool(host='nearfinderza.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nearfinderza.com', port=443): Read timed out. (read timeout=2)"
neaselida.gr,"HTTPSConnectionPool(host='neaselida.gr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='neaselida.gr', port=80): Read timed out. (read timeout=2)"
neliti.com,"HTTPSConnectionPool(host='neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
neopod.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='neopod.ru', port=80): Read timed out. (read timeout=2)"
neowing.co.jp,"HTTPSConnectionPool(host='neowing.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bdfaab20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='neowing.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdbd81f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
net.ee,"HTTPSConnectionPool(host='net.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bddaac10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='net.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdb837c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
neturuguay.com,"HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)"
newbalkan.cz,"HTTPSConnectionPool(host='newbalkan.cz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bdd14730>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='newbalkan.cz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdd48a30>: Failed to establish a new connection: [Errno 111] Connection refused'))"
newellwind.com,"HTTPSConnectionPool(host='newellwind.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bdf3cc10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='newellwind.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdfb1280>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newenglandsquash.com,"HTTPSConnectionPool(host='newenglandsquash.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bdd48100>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='newenglandsquash.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdbf01f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newpages.com.my,"HTTPSConnectionPool(host='www.newpages.com.my', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.newpages.com.my', port=443): Read timed out. (read timeout=2)"
news-italia-mondo.it,Exceeded 30 redirects.,Exceeded 30 redirects.
newsbomb.com.cy,"HTTPSConnectionPool(host='newsbomb.com.cy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'newsbomb.com.cy' doesn't match either of '*.constellix.net', 'constellix.net'"")))","HTTPConnectionPool(host='www.newsbomb.com.cy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bd9bd160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newsonair.com,"HTTPSConnectionPool(host='newsonair.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='newsonair.com', port=443): Read timed out. (read timeout=2)"
newtv.co.th,"HTTPSConnectionPool(host='newtv.co.th', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='newtv.co.th', port=80): Read timed out. (read timeout=2)"
newzealandnow.govt.nz,Exceeded 30 redirects.,Exceeded 30 redirects.
nguoiduatin.vn,"HTTPSConnectionPool(host='nguoiduatin.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bc546b50>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nguoiduatin.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f62bd42bfd0>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))"
nhk.or.jp,"HTTPSConnectionPool(host='nhk.or.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bc052e50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nhk.or.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bce1c160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ni.rs,"HTTPSConnectionPool(host='ni.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bc76f760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ni.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bc530fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nichepoliambulatorio.it,Exceeded 30 redirects.,Exceeded 30 redirects.
nii.ac.jp,"HTTPSConnectionPool(host='nii.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bb8e34f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nii.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdcc8100>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
niigata.lg.jp,"HTTPSConnectionPool(host='niigata.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bb9c7160>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='niigata.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bd4e3fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nikib.co.il,"HTTPSConnectionPool(host='nikib.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nikib.co.il', port=443): Read timed out. (read timeout=2)"
nikkei.co.jp,"HTTPSConnectionPool(host='nikkei.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bc34aaf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nikkei.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bc42c0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ninhbinh.gov.vn,"HTTPSConnectionPool(host='ninhbinh.gov.vn', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bdfb1640>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='ninhbinh.gov.vn', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdc789d0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
ninhthuan.gov.vn,"HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)"
nippon.com,"HTTPSConnectionPool(host='nippon.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bd97b820>, 'Connection to nippon.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nippon.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f62bdbf0190>, 'Connection to nippon.com timed out. (connect timeout=2)'))"
njuzz.ee,"HTTPSConnectionPool(host='njuzz.ee', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bdb8ecd0>, 'Connection to njuzz.ee timed out. (connect timeout=2)'))","HTTPConnectionPool(host='njuzz.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bd50ce50>: Failed to establish a new connection: [Errno 113] No route to host'))"
nojima.co.jp,"HTTPSConnectionPool(host='nojima.co.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bb86f880>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nojima.co.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f62bb90a3d0>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))"
nomos-elibrary.de,"HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)"
nordicchoicehotels.no,"HTTPSConnectionPool(host='www.nordicchoicehotels.no', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nordicchoicehotels.no', port=443): Read timed out. (read timeout=2)"
norskposten.com,"HTTPSConnectionPool(host='norskposten.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='norskposten.com', port=80): Read timed out. (read timeout=2)"
noticiasdecoimbra.pt,"HTTPSConnectionPool(host='noticiasdecoimbra.pt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='noticiasdecoimbra.pt', port=443): Read timed out. (read timeout=2)"
novine.hr,"HTTPSConnectionPool(host='novine.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='novine.hr', port=443): Read timed out. (read timeout=2)"
novosti.hr,"HTTPSConnectionPool(host='novosti.hr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='novosti.hr', port=80): Read timed out. (read timeout=2)"
novynabytok.sk,"HTTPSConnectionPool(host='www.novynabytok.sk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.novynabytok.sk', port=443): Read timed out. (read timeout=2)"
npru.ac.th,"HTTPSConnectionPool(host='npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
npz.sk,"HTTPSConnectionPool(host='npz.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
nra.bg,"HTTPSConnectionPool(host='nra.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nra.bg', port=443): Read timed out. (read timeout=2)"
nrsr.sk,"HTTPSConnectionPool(host='nrsr.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bddb3b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nrsr.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bb7d2a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ns.ac.rs,"HTTPSConnectionPool(host='ns.ac.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bd84d880>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ns.ac.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bb7d26a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nsc.ru,"HTTPSConnectionPool(host='nsc.ru', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bddaaac0>, 'Connection to nsc.ru timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nsc.ru', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f62bb915c70>, 'Connection to nsc.ru timed out. (connect timeout=2)'))"
nsk.hr,"HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)"
nso.go.th,"HTTPSConnectionPool(host='nso.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bc35ab20>, 'Connection to nso.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nso.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f62bc39ba60>, 'Connection to nso.go.th timed out. (connect timeout=2)'))"
nstru.ac.th,"HTTPSConnectionPool(host='nstru.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bd9bd8e0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='nstru.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdfe77c0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
nta.go.jp,"HTTPSConnectionPool(host='nta.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bbea3b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nta.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bb8c3ca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ntn24.com,"HTTPSConnectionPool(host='ntn24.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1124)')))","HTTPSConnectionPool(host='ntn24.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1124)')))"
nttdocomo.co.jp,"HTTPSConnectionPool(host='nttdocomo.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bdd3b070>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nttdocomo.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bdfb1250>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ntua.gr,"HTTPSConnectionPool(host='ntua.gr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bda12730>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='ntua.gr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bc76ff10>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
nucleodoconhecimento.com.br,"HTTPSConnectionPool(host='nucleodoconhecimento.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nucleodoconhecimento.com.br', port=443): Read timed out. (read timeout=2)"
number1.co.il,"HTTPSConnectionPool(host='number1.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='number1.co.il', port=80): Read timed out. (read timeout=2)"
nusalist.com,"HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)"
nuv.cz,"HTTPSConnectionPool(host='nuv.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPConnectionPool(host='www.nuv.cz', port=80): Read timed out. (read timeout=2)"
nwu.ac.za,"HTTPSConnectionPool(host='nwu.ac.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bc052e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nwu.ac.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bda91790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nydailynews.com,"HTTPSConnectionPool(host='nydailynews.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1124)')))","HTTPConnectionPool(host='www.nydailynews.com', port=80): Read timed out. (read timeout=2)"
nyf.hu,"HTTPSConnectionPool(host='nyf.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bb9973a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nyf.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bd3378b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nzlii.org,"HTTPSConnectionPool(host='nzlii.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='nzlii.org', port=80): Read timed out. (read timeout=2)"
nzt.lt,"HTTPSConnectionPool(host='nzt.lt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f62bdf7c250>, 'Connection to nzt.lt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nzt.lt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f62bc5c6dc0>, 'Connection to nzt.lt timed out. (connect timeout=2)'))"
obraspublicas.gob.ec,"HTTPSConnectionPool(host='obraspublicas.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'obraspublicas.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
observador.cl,"HTTPSConnectionPool(host='observador.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bc0b6490>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='observador.cl', port=80): Read timed out. (read timeout=2)"
obz.hr,"HTTPSConnectionPool(host='obz.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f62bb8a1310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='obz.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f62bc014f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
